.github/workflows/ci.yml

```diff
@@ -742,7 +742,7 @@ jobs:
       ICEBERG_MAIN_REPOSITORY: apache/iceberg
       ICEBERG_MAIN_BRANCH: master
       ICEBERG_PATCH_REPOSITORY: snazy/iceberg
-      ICEBERG_PATCH_BRANCH: iceberg-nesqueit
+      ICEBERG_PATCH_BRANCH: iceberg-nesqueit-err
```

Comment:
Do we still need a custom iceberg branch?

Reply:
Yea :(